### ISSUE TITLE
Allow users to take pictures from the app

### ISSUE THREAD

**Issue by WERUreo** [cforlando/orlando-walking-tours-ios#43]
The user might want to take a picture of each location right from within the app.

**Comment by akozlik** [comment]
Same question about MVP here. How are we benefitting from allowing users to take photos from the app? Are they going to submit the same photo over and over again? It may be better just to remove this requirement, especially for MVP.

**Comment by WERUreo** [comment]
Yeah, this might be one of those that gets removed once we start incorporating social networking like Instagram.

**Comment by akozlik** [comment]
Sounds like a plan.
